docs/source/installation.rst

```diff
@@ -2,147 +2,123 @@ Installation
 
-Open your bash/zsh script file with editor on your computer.
+Step 1: Setup dedicated python environment
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
 
-    If you are using bash shell
-    ::
+You can setup ``ivadomed`` using either Conda or Venv:
 
-        vim ~/.bashrc
+.. tabs::
 
-    If you are using zsh shell
-    ::
-        
-        vim ~/.zshrc
+    .. tab:: Install via ``conda``
 
-Write in your .bashrc/.zshrc file with following line.
+        This is the easiest way for personal computers.
 
-::
+        1. Create new conda environment using ``environment.yml`` file
 
-    export HDF5_USE_FILE_LOCKING='FALSE'
+        ::
 
-Save this change and restart your terminal to apply the change.
+            conda env create --file environment.yml
 
-There are fundamentally two different approaches to install ``IvadoMed``:
+        2. Activate the new conda environment (default named ``ivadomed_env``)
 
-1) Install via Conda
-    This is the easiest way for personal computers.
+        ::
 
-2) Install via Venv/VirtualEnv
-    This is compatible with ComputeCanada cluster environment.
+            conda activate ivadomed_env
 
-Approach 1: Conda
-------------------
+    .. tab:: Install via ``venv``
 
-Step 1: Clone the `ivadomed <https://github.com/ivadomed/ivadomed>`_ repository.
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+        1. Setup Python Venv Virtual Environment.
 
-::
+        ``ivadomed`` requires Python >= 3.6 and <3.9 (If you are using `Compute Canada <https://www.computecanada.ca/>`_, you can load modules (e.g. python 3.9) as `mentioned here <https://intranet.neuro.polymtl.ca/computing-resources/compute-canada#modules>`_ and `also here <https://docs.computecanada.ca/wiki/Utiliser_des_modules/en#Loading_modules_automatically>`_ ). We recommend
+        working under a virtual environment, which could be set as follows:
 
-    git clone https://github.com/neuropoly/ivadomed.git
-    cd ivadomed
+        ::
 
-Step 2: Create new Conda Env called IvadoMedEnv (may take 5 to 15 minutes)
-+++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++
+            python -m venv ivadomed_env
 
-::
+        2. Activate the new virtual environment (default named `ivadomed_env`)
 
-    conda env create --file environment.yml
+        ::
 
-Step 3 : Activate environment and use
-++++++++++++++++++++++++++++++++++++++
+            source ivadomed_env/bin/activate
 
-::
 
-    conda activate IvadoMedEnv
+        .. warning::
+           If the default Python version installed in your system does not fit the version requirements, you might need to specify a version of Python associated with your virtual environment:
 
-Step 4 : Install from source
-++++++++++++++++++++++++++++++++++++++
+           ::
 
-::
+             python -m venv ivadomed_env --python=python3.7
```

Comment:
I've never seen this `-python=` syntax... and when I try it, I get an error:

```
joshua@tadpole:~$ python -m venv ivadomed_env --python=python3.7
usage: venv [-h] [--system-site-packages] [--symlinks | --copies] [--clear]
            [--upgrade] [--without-pip] [--prompt PROMPT] [--upgrade-deps]
            ENV_DIR [ENV_DIR ...]
venv: error: unrecognized arguments: --python=python3.7
```

Just curious, where did you find this? o:

Reply:
Oh man... VENV doesn't support this :(. Okay. Will have to remove this section then and hope they use the right python version to begin with.

Reply:
Thankfully, the range of versions that ivadomed supports is quite wide, so I think in the vast majority of cases, the default Python will be OK.

Reply:
Ah, this has a better solution in https://github.com/ivadomed/ivadomed/pull/969#discussion_r744307483.